app/code/Magento/InventoryInStorePickup/etc/db_schema.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="inventory_pickup_point_order" resource="default" engine="innodb">
```

Comment:
We have to discuss `Pickup Point` naming. It's best to align it with business domain

Reply:
Changed to 'Pickup Location'.